mlx/backend/metal/kernels/quantized.metal

```diff
@@ -1,1516 +1,50 @@
 // Copyright © 2023-2024 Apple Inc.
 
-#include <metal_simdgroup>
-#include <metal_stdlib>
-
 #include "mlx/backend/metal/kernels/bf16.h"
 #include "mlx/backend/metal/kernels/defines.h"
+#include "mlx/backend/metal/kernels/quantized.h"
```

Comment:
Tests are failing probably because these are in the wrong order. You can put a clang format off at the top of this file to keep the pre-commit hooks from changing them. I would just turn it off at the top and turn it back on at the bottom (the clang formatting is usually not so useful for these preprocessor files anyway and often messes it up)